shared/domain/src/main/java/gov/va/vro/model/AbdEvidenceWithSummary.java

```diff
@@ -27,5 +27,5 @@ public class AbdEvidenceWithSummary {
 
   @Schema(description = "Sufficient for Fast Tracking flag")
   @JsonInclude(JsonInclude.Include.NON_EMPTY)
-  private Boolean sufficientForFastTracking;
+  private boolean sufficientForFastTracking;
```

Comment:
This is a required value. If the service does not return it, we have to default to "false"